### DISCUSSION TITLE
Error failed to fetch an image or build from source

### DISCUSSION THREAD

**Discussion by jayvzh** [York618/alist-flyio#2] [General]
https://github.com/jayvzh/alist-flyio/runs/7000043932?check_suite_focus=true#step:3:110

> Step 6/6 : CMD /***.sh
 ---> Running in 66ed15a16fbd
 ---> c1913c38f45c
Successfully built c1913c38f45c
Successfully tagged registry.fly.io/***:deployment-1655885976
--> Building image done
==> Pushing image to fly
The push refers to repository [registry.fly.io/***]
[92](https://github.com/jayvzh/alist-flyio/runs/7000043932?check_suite_focus=true#step:3:93)d487afdf2e: Preparing
4[94](https://github.com/jayvzh/alist-flyio/runs/7000043932?check_suite_focus=true#step:3:95)74313bb21: Preparing
1460ff244bf0: Preparing
c46e27[96](https://github.com/jayvzh/alist-flyio/runs/7000043932?check_suite_focus=true#step:3:97)92cf: Preparing
e4058d818b56: Preparing
48f27885bee7: Preparing
48f27885bee7: Waiting
Error failed to fetch an image or build from source: error rendering push status stream: unauthorized: not allowed
Error: Process completed with exit code 1.

这是什么问题 是不是alist的图片出问题了 之前vps部署时候就有问题，alist的好多图片都挂了

**Comment by York618** [comment]
很抱歉由于现实的种种原因，现在才能回复您
刚才调取日志可以看到
`Error You must be authenticated to view this. `
这边建议您检查Fly.io的API_TOKEN有效性（未添加/过期等）

> FLY_API_TOKEN - Fly API 接口 Token 值，可访问 https://web.fly.io/user/personal_access_tokens 或在本地执行flyctl auth token查看
